environmentmodel/environment-model_shacl.ttl

```diff
@@ -0,0 +1,288 @@
+    sh:targetClass environmentmodel:georeference .
+
+environmentmodel:GeodeticReferenceSystemShape a sh:NodeShape ;
+    sh:property [ skos:example "Lat: 1.234; Lon 45.23" ;
+            sh:description "World coordinates of map origin"@en ;
+            sh:message "Validation of origin failed!"@en ;
+            sh:name "origin"@en ;
+            sh:node environmentmodel:Coordinate2DShape ;
+            sh:minCount 1 ;
+            sh:maxCount 1 ;
+            sh:order 2 ;
+            sh:path environmentmodel:origin ],
+        [ skos:example "EPSG 32632" ;
+            sh:datatype xsd:string ;
+            sh:description "EPSG code of the map"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of coordinateSystem failed!"@en ;
+            sh:name "coordinateSystem"@en ;
+            sh:order 0 ;
+            sh:path environmentmodel:coordinateSystem ],
+        [ skos:example "EPSG xyz" ;
+            sh:datatype xsd:string ;
+            sh:description "Ellipsodial height or orthometric height"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of heightSystem failed!"@en ;
+            sh:name "heightSystem"@en ;
+            sh:order 1 ;
+            sh:path environmentmodel:heightSystem ] ;
+    sh:targetClass environmentmodel:GeodeticReferenceSystem .
+
+environmentmodel:BoundingBoxShape a sh:NodeShape ;
+    sh:property [ sh:datatype xsd:float ;
+            sh:message "Validation of yMin failed!"@en ;
+            sh:name "yMin"@en ;
+            sh:maxCount 1 ;
+            sh:order 1 ;
+            sh:path environmentmodel:yMin ],
+        [ sh:datatype xsd:float ;
+            sh:message "Validation of yMax failed!"@en ;
+            sh:name "yMax"@en ;
+            sh:maxCount 1 ;
+            sh:order 3 ;
+            sh:path environmentmodel:yMax ],
+        [ sh:datatype xsd:float ;
+            sh:message "Validation of xMin failed!"@en ;
+            sh:name "xMin"@en ;
+            sh:maxCount 1 ;
+            sh:order 0 ;
+            sh:path environmentmodel:xMin ],
+        [ sh:datatype xsd:float ;
+            sh:message "Validation of xMax failed!"@en ;
+            sh:name "xMax"@en ;
+            sh:maxCount 1 ;
+            sh:order 2 ;
+            sh:path environmentmodel:xMax ] ;
+    sh:targetClass environmentmodel:BoundingBox .
+
+environmentmodel:Coordinate2DShape a sh:NodeShape ;
+    sh:property [ sh:datatype xsd:float ;
+            sh:message "Validation of x failed!"@en ;
+            sh:name "x"@en ;
+            sh:maxCount 1 ;
+            sh:order 0 ;
+            sh:path environmentmodel:x ],
+        [ sh:datatype xsd:float ;
+            sh:message "Validation of y failed!"@en ;
+            sh:name "y"@en ;
+            sh:maxCount 1 ;
+            sh:order 1 ;
+            sh:path environmentmodel:y ] ;
+    sh:targetClass environmentmodel:Coordinate2D .
+    
+environmentmodel:ProjectLocationShape a sh:NodeShape ;
+    sh:property [ skos:example "nan" ;
```

Comment:
if there is no valid example I would remove the example with "nan"

Please correct this for the others too

Reply:
@MircoNierenz can you look into that topic? I think currently it is a placeholder.
@robertschubert can merge this branch allready into main with this placeholder?

Reply:
I think we can merge without having a solution right now.
Opened Issue https://github.com/GAIA-X4PLC-AAD/ontology-management-base/issues/36